client/components/site-stats-sticky-link/index.jsx

```diff
@@ -36,11 +39,23 @@ var SiteStatsStickyLink = React.createClass( {
 		}
 	},
 
+	preload: function() {
```

Comment:
For now this looks great, but I would agree that we should put some more thought into this before letting the pattern spread.